src/track/beatmap.h

```diff
@@ -80,6 +80,14 @@ class BeatMap final : public Beats {
     void scale(enum BPMScale scale) override;
     void setBpm(double dBpm) override;
 
+    ////////////////////////////////////////////////////////////////////////////
```

Comment:
We do not use ascii borders.

Reply:
Hummm. I don't specially like the use of this kind of ascii art. But the rest of the file is written like that so I was just copying the stile.
I will take that into account while working on the new Beats class.

Reply:
Done